semcore/utils/src/components/invalid-state-pattern/invalidStatePattern.shadow.css

```diff
@@ -0,0 +1,10 @@
+SPattern {
+  width: 6px;
+  height: 100%;
+  position: absolute;
+  background-image: repeating-linear-gradient(315deg,var(--intergalactic-border-critical-active, #D1002F) 0,var(--intergalactic-border-critical-active, #D1002F) 2px,transparent 0,transparent 50%);
```

Comment:
I'll add a token for this with the name `--intergalactic-border-critical-pattern`.